tools/donate_cpu_lib.py

```diff
@@ -370,21 +375,20 @@ def scan_package(work_path, cppcheck_path, jobs, libraries):
         if has_sig:
             returncode = -sig_num
         stacktrace = ''
-        if cppcheck_path == 'cppcheck':
```

Comment:
We still need that check since the traces are only supposed to be generated for the `main`/`head` case and not the tags.